lib/clever-ruby/district_admin.rb

```diff
@@ -0,0 +1,17 @@
+module Clever
+  # District Admin resource
+  class DistrictAdmin < APIResource
+    include Clever::APIOperations::List
+    @uri = 'district_admins'
+    @plural = 'district_admins'
+    @linked_resources = []
+
+    # Optional attributes
+    # @see Clever::CleverObject.optional_attributes
+    # @api private
+    # @return [Array]
+    def optional_attributes
+      [:name, :title]
```

Comment:
I was wondering about this myself.  I think it makes more sense to trust whatever the API documentation / schema on their website says, rather than what's in the gem.  My guess is this is either an oversight w/ whoever wrote the gem initially, or their API changed from requiring name to requiring email.  I was thinking of submitting a PR to the official repo fixing that as well, but that's for another day